miio/vacuum_tui.py

```diff
@@ -0,0 +1,95 @@
+try:
+    import curses
+except ImportError:
+    pass
+
+import enum
+from typing import Tuple, Type
+
+from .vacuum import Vacuum
+
+
+class Control(enum.Enum):
+
+    Quit = "q"
+    Forward = "l"
+    ForwardFast = "L"
+    Backward = "k"
+    BackwardFast = "K"
+    Left = "j"
+    LeftFast = "J"
+    Right = ";"
+    RightFast = ":"
```

Comment:
Rather than using obscure key bindings, how about using the arrow keys & shift as a modifier for the fast modes?

Reply:
Current bindings are Vim-ish.

Arrow keys work fine but lack support for the fast mode. The reason is that for `curses` there are no such things as Shift+ArrowUp or Shift+ArrowDown (Shift+ArrowLeft and Shift+ArrowRight are OK though).

Shall go with WASD?

Reply:
Hmm, I think WASD is a better choice (as `;` and `:` can be problematic on some keyboard layouts), not that everything is QWERTY either, but it's still a commonly used control scheme...

Reply:
Alright, changed key bindings to WASD.